R/geom-point.R

```diff
@@ -85,6 +85,13 @@
 #' ggplot(mtcars, aes(wt, mpg)) +
 #'   geom_point(shape = 21, colour = "black", fill = "white", size = 5, stroke = 5)
 #'
+#' # The default shape used in legends is not filled, so you will need to
+#' # override it in the guide in order to see the fill reflected in the legend
```

Comment:
```suggestion
#' # The default shape in legends is not filled, but you can
#' # override the shape in the guide to reflect the fill
```
Just slightly more terse